### ISSUE TITLE
cmdapi: support push requests

### ISSUE THREAD

**Issue by jessepeterson** [micromdm/micro2nano#3]
`cmdapi` only supports the, well, command API. Consider also supporting proxying the MicroMDM push API as well. I think @discentem may have requested this feature at some point, too.

**Comment by discentem** [comment]
Yeah this would be great! I started work to proxy some other stuff in micromdm that mdmdirector needs but haven't worked on that in a while: https://github.com/discentem/nanomdmsandbox/blob/devices_handler/cmd/devices/main.go
